### ISSUE TITLE
Vo and Content card do not match

### ISSUE THREAD

**Issue by ashengit2025** [oppia/lessons-team#1884]
Place Values -> Finding the value of a number
Device: Motorola (Android)
Testing done on app
In 'Finding the place value of a number' chapter, the voiceover  says **2 millions** + 6 ten thousands + 3 tens + 8 ones where as in the content card it only shows 6 ten thousands + 3 tens + 8 ones (the 2 millions is missing).
Screenshot of the the lesson attached

<img width="1080" height="2160" alt="Image" src="https://github.com/user-attachments/assets/8fdfcebe-ba11-4329-978e-0cea4644fc2f" />
